core/src/test/kotlin/org/evomaster/core/output/TestCaseWriterTest.kt

```diff
@@ -15,15 +15,17 @@ import org.evomaster.core.output.service.RestTestCaseWriter
 import org.evomaster.core.problem.rest.*
```

Comment:
can u revert the formatting here? with all the formatting changes, it is hard to see which are the new modifications...